sql/moz-fx-data-marketing-prod/ga_derived/firefox_whatsnew_summary_v1/query.sql

```diff
@@ -0,0 +1,34 @@
+WITH wnp_visits AS (
+  SELECT
+    date,
+    visit_identifier,
+    hit_number,
+    bounces,
+    first_interaction,
```

Comment:
The `hit_number`, `bounces`, and `first_interaction` output columns of this CTE are no longer used and can be removed.
```suggestion
```